relay-server/src/actors/processor.rs

```diff
@@ -1274,6 +1282,77 @@ impl EnvelopeProcessorService {
         }
     }
 
+    fn handle_submit_envelope(&self, message: SubmitEnvelope) {
+        let SubmitEnvelope { mut envelope } = message;
+
+        #[cfg(feature = "processing")]
+        if self.inner.config.processing_enabled() {
+            if let Some(store_forwarder) = self.inner.store_forwarder.clone() {
+                relay_log::trace!("sending envelope to kafka");
+                store_forwarder.send(StoreEnvelope { envelope });
```

Comment:
The store forwarder used to return an async response with a potential error. Now it no longer has to be awaited, and instead logs outcomes internally through the managed envelope.